### ISSUE TITLE
Inference Problem with DeepSpeech2

### ISSUE THREAD

**Issue by jfornaro** [TensorSpeech/TensorFlowASR#278]
Hi,
I am having difficultis with DeepSpeech2 inference. I train with batch_size=64. In training, the loss goes down as expected. When I do inference with the test.py example, I get very bad results when I set batch_size=1 but acceptable results when batch_size=4 or batch_size=64. Could you please point me in the right direction?
